MeNext/Layout/LibraryView.cs

```diff
@@ -53,15 +53,21 @@ public LibraryView(MainController mainController)
             this.mainController.Event.RegisterUiListener(this);
             this.SomethingChanged();
         }
-
+        /// <summary>
+        /// User's Spotify account's login status has changed.
```

Comment:
Remove this line. This function is called whenever anything pertaining to the UI changes.